### PR TITLE
add hw03

### PR DESCRIPTION
**Pull request by Naidantu**

### CODE REVIEW

**Comment by Naidantu** [comment]
@gregrousis @whitneykasenetz

File: hw03/hw03.Rmd [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+     
+```{r}
+as_tibble(green_data)
+green_data %>% 
+  summarize(across(c(green1:student), list(mean= ~ mean(.x, na.rm=T), sd= ~sd(.x, na.rm = T), 
+                                      min= ~min(.x, na.rm = T), max= ~max(.x, na.rm = T))))
+```
```
Comment (gregrousis) [inline hw03/hw03.Rmd]: here it might be easier to just summarize all the data using 
`green_data %>% 
  summary()`

File: hw03/hw03.Rmd [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+green_data <- green_data %>% 
+  rowwise() %>% 
+  mutate(green=sum(c_across(green1:green5), na.rm = T),
+         comp=sum(c_across(comp1:comp10), na.rm = T),
+         intel=sum(c_across(intel1:intel10), na.rm = T),
+         open=sum(c_across(open1:open10), na.rm = T)) 
+
+green_data[green_data == 0] <- NA
+```
```
Comment (gregrousis) [inline hw03/hw03.Rmd:76]: where were the 0s in your dataset? Were they in the computed scales?

File: hw03/hw03.Rmd [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+  3. Rescale the variables so that they go from 0-100 instead of the original range.
+     Name the recaled variables `*_pomp`.
+
+```{r}
+green_pomp=((green_data$green-min(green_data$green, na.rm = T))/(max(green_data$green, na.rm = T)-min(green_data$green, na.rm = T)))*100
+comp_pomp=((green_data$comp-min(green_data$comp, na.rm = T))/(max(green_data$comp, na.rm = T)-min(green_data$comp, na.rm = T)))*100
+intel_pomp=((green_data$intel-min(green_data$intel, na.rm = T))/(max(green_data$intel, na.rm = T)-min(green_data$intel, na.rm = T)))*100
+open_pomp=((green_data$open-min(green_data$open, na.rm = T))/(max(green_data$open, na.rm = T)-min(green_data$open, na.rm = T)))*100
+green_data1 <- cbind(green_data, green_pomp, comp_pomp, intel_pomp, open_pomp)
```
Comment (gregrousis) [inline hw03/hw03.Rmd]: I think converting these to POMP might be best if you supply the minimum and maximum possible values. Here your min and max values are based on what the participants responded as opposed to the minimum and maximum possible values. That will likely restrict the range of your variables.

File: hw03/hw03.Rmd [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+```{r}
+green_data1 %>% 
+  filter(!is.na(student)) %>% 
+    group_by(student) %>%
+    summarize(across(c(green_pomp:open_pomp), list(mean= ~ mean(.x, na.rm=T), sd= ~sd(.x, na.rm = T), 
+                                      median= ~median(.x, na.rm = T), min= ~min(.x, na.rm = T), max= ~max(.x, na.rm = T))))
+```  
+  
+In your assignment, prepare an RMarkdown file that includes both the coding and (for 4-7) narrative descriptions of your results.
```
Comment (gregrousis) [inline hw03/hw03.Rmd:182]: Nice job Naidan!!!

**Review by gregrousis — COMMENTED** [review]
Great job!!

File: hw03/hw03.Rmd (hunk 2 above)
Comment (Naidantu) [reply hw03/hw03.Rmd:76]: Yes, the NAs in the original items become 0 in the total scores

File: hw03/hw03.Rmd (hunk 3 above)
Comment (Naidantu) [reply hw03/hw03.Rmd]: You're right! I just changed the code

**Pull request merged**